jormungandr-lib/src/time.rs

```diff
@@ -273,15 +273,13 @@ impl From<u64> for SystemTime {
     }
 }
 
-impl From<u64> for SecondsSinceUnixEpoch {
-    fn from(seconds_since_epoch: u64) -> Self {
+impl SecondsSinceUnixEpoch {
+    pub fn from_secs_since_epoch(seconds_since_epoch: u64) -> Self {
         SecondsSinceUnixEpoch(seconds_since_epoch)
     }
-}
 
-impl From<SecondsSinceUnixEpoch> for u64 {
-    fn from(seconds_since_epoch: SecondsSinceUnixEpoch) -> Self {
-        seconds_since_epoch.0
+    pub fn into_secs_since_epoch(self: SecondsSinceUnixEpoch) -> u64 {
```

Comment:
The `.into()` part was actually fine, as there is no precondition to observe during conversion to plain integer. I'd leave it as the standard conversion trait.

Reply:
Now it's obvious what's the meaning of the output number. The integer could be anything, e.g. miliseconds since epoch, or seconds since startup, but now it's clearly seconds since epoch.

Reply:
Renamed to `to_secs`